keyboards/handwired/lagrange/config.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2020 Dimitris Papavasiliou <dpapavas@protonmail.ch>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x2718
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Dimitris Papavasiliou
+#define PRODUCT         Lagrange
+#define DESCRIPTION     An ergonomic, split keyboard with thumb clusters.
+
+#define EE_HANDS
+#define SPLIT_USB_DETECT
+
+/* key matrix size */
+#define MATRIX_ROWS 14
+#define MATRIX_COLS 6
+
+/* pin-out */
+#define MATRIX_ROW_PINS { E6, F1, F0, F4, F5, F6, F7 }
+#define MATRIX_COL_PINS { B4, B5, D7, B6, C6, D6 }
+#define MATRIX_ROW_PINS_RIGHT { B5, B4, D7, B6, C6, D6, D4 }
+#define MATRIX_COL_PINS_RIGHT { C7, F7, F6, F5, F4, F1 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION ROW2COL
+
+#define DEBOUNCE_TYPE = sym_defer_pk
```

Comment:
this would need to go into the rules.mk, not here. 
```suggestion
```

Additionally, this is already the default behavior.

Reply:
The [documentation](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_debounce_type.md#debounce-selection) suggests `sym_defer_g` is the default and it seems to be the the case in [the code](https://github.com/qmk/qmk_firmware/blob/c66df1664497546f32662409778731143e45a552/common_features.mk#L440) as well.

I'll just move it to `rules.mk`.